cli/go.mod

```diff
@@ -142,3 +143,5 @@ require (
 
 // github.com/cloudquery/jsonschema @ cqmain
 replace github.com/invopop/jsonschema => github.com/cloudquery/jsonschema v0.0.0-20231018073309-6c617a23d42f
+
+replace github.com/cloudquery/cloudquery-api-go => github.com/cloudquery/cloudquery-api-go v1.6.4-0.20240104155404-515b000bd747
```

Comment:
Using untagged for now

Reply:
what's the reason for this? let's release the next version if that's what you need

Reply:
the reason was being unsure if anything else (like content type) was needed in the request :) will release shortly.